FalmouthFalcons/iteration-3
# Clayton

Description:
-Add description about what this pull request covers.
Added Shopping Cart Functionality Pull Down and make sure after you view a product you can add it to you cart and image shows up
Number of Fixes:
-Type of number of fixes included in this PR.
0
Related Ticket(s):
- List related ticket(s) that apply to the work done in this pull request
#12 
Problem to Solve:
- Describe the problem(s) this pull request solves.

Proposed Changes:
-Describe the proposed changes.

Expected Behavior:
- Describe the expected behavior.

Steps to Test Solution
1) git pull branch clayton 
2)git fetch 
3)test

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
